### PR TITLE
Update is_supported() in memconfig.py

### PR DESCRIPTION
**Pull request by frinzell**
- `is_supported()` doesn't need to check for 'MSR_BIOS_DONE'
- Some clean up

Signed-off-by: Frinzell, Aaron <aaron.frinzell@intel.com>

### CODE REVIEW

File: chipsec/modules/common/memconfig.py [hunk 1]
```diff
@@ -93,18 +93,20 @@ def check_memmap_locks(self):
...
-            d = self.cs.get_register_def(r)
-            v = self.cs.read_register(r)
-            locked = self.cs.get_register_field(r, v, memmap_registers[r])
+            reg_def = self.cs.get_register_def(reg)
+            reg_value = self.cs.read_register(reg)
+            reg_desc = reg_def['desc']
+            locked = self.cs.get_register_field(reg, reg_value, reg_field)
             if locked == 1:
-                self.logger.log_good("{:20} = 0x{:16X} - LOCKED   - {}".format(r, v, d['desc']))
+                self.logger.log_good(f"{reg:20} = 0x{reg_value:016X} - LOCKED   - {reg_desc}")
```
Comment (dscott90) [inline chipsec/modules/common/memconfig.py:106]:
```suggestion
                self.logger.log_good(f'{reg:20} = 0x{reg_value:016X} - LOCKED   - {reg_desc}')
```

File: chipsec/modules/common/memconfig.py [hunk 2]
```diff
@@ -93,18 +93,20 @@ def check_memmap_locks(self):
...
             else:
                 all_locked = False
-                self.logger.log_bad("{:20} = 0x{:16X} - UNLOCKED - {}".format(r, v, d['desc']))
+                self.logger.log_bad(f"{reg:20} = 0x{reg_value:016X} - UNLOCKED - {reg_desc}")
```
Comment (dscott90) [inline chipsec/modules/common/memconfig.py:109]:
```suggestion
                self.logger.log_bad(f'{reg:20} = 0x{reg_value:016X} - UNLOCKED - {reg_desc}')
```

**Review by dscott90 — APPROVED** [review]

File: chipsec/modules/common/memconfig.py [hunk 3]
```diff
@@ -93,18 +93,20 @@ def check_memmap_locks(self):
             self.logger.log('[*] Checking legacy register lock state:')
         else:
             self.logger.log('[*] Checking register lock state:')
-        for r in regs:
-            if not self.cs.is_register_defined(r) or not self.cs.register_has_field(r, memmap_registers[r]):
-                self.logger.log_important('Skipping Validation: Register {} or field {} was not defined for this platform.'.format(r, memmap_registers[r]))
+        for reg in regs:
+            reg_field = memmap_registers[reg]
+            if not self.cs.register_has_field(reg, reg_field):
+                self.logger.log_important(f'Skipping Validation: Register {reg} or field {reg_field} was not defined for this platform.')
```
Comment (npmitche) [inline chipsec/modules/common/memconfig.py:99]:
```suggestion
                self.logger.log_important(f"Skipping Validation: Register {reg} or field {reg_field} was not defined for this platform.")
```
We should be consistent, but I think we should go with double quotes. `"`

**Review by npmitche — APPROVED** [review]

**Comment by npmitche** [comment]
Requires https://github.com/chipsec/chipsec/pull/1567

**Pull request merged**